### PR TITLE
Adds all-seeing-eye to necropolis chests.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
This replaces the xray mutation syringe. 

fuck

### CODE REVIEW

**Comment by coiax** [comment]
It should really just burrow into the skull of the person wearing it and given them xray vision as a result (and maybe changing their eye colour constantly?). Could basically be xray cyberimplant to accomplish that.

**Comment by Kromgar** [comment]
So... the eye of Vecna?


We need to add the hand of Vecna. IF you have both you can shoot lasers out your eyes

**Comment by PKPenguin321** [comment]
We should make eyes into an organ first and add this as an organ

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -279,6 +279,16 @@
...
 
+/obj/item/clothing/glasses/godeye
+	name = "eye of god"
+	desc = "A strange eye, said to have been torn from an omniscient creature that used to roam the wastes."
+	icon_state = "godeye"
+	item_state = "godeye"
+	vision_flags = SEE_TURFS|SEE_MOBS|SEE_OBJS
+	darkness_view = 8
+	scan_reagents = 1
+	flags = 2
```
Comment (phil235) [inline code/modules/clothing/glasses/glasses.dm]: no define?

**Comment by TechnoAlchemisto** [comment]
I have no idea how to code

File: code/modules/clothing/glasses/glasses.dm (hunk 1 above)
Comment (phil235) [inline code/modules/clothing/glasses/glasses.dm]: `flags = NODROP` since 2 is NODROP.

ALthough are you sure you want the item to be not be unequippable?

Comment (TechnoAlchemisto) [inline code/modules/clothing/glasses/glasses.dm]: Ah, yes, I'd like for it to be stuck to the person's face. I've always liked the theme of cursed items on lavaland.

**Comment by Alek2ander** [comment]
http://www.blindpanic.com/humor/vecna.htm

**Comment by coiax** [comment]
Techno, you realise that NODROP means that when they pick it up,  they won't be able to move it to their eye slot, it'll just stay stuck to their hand, like the chaplain's  godhand?

**Comment by KorPhaeron** [comment]
You can definitely pick up and equip no drop hats at least

**Comment by TechnoAlchemisto** [comment]
I tested it, you can pick it up (can't drop it) , but they can equip it to their eye and it stays there. It's REAL STICKY.

**Pull request merged**